pkg/cmd/init.go

```diff
@@ -43,9 +44,12 @@ These resources can be removed from the cluster using the "cleanup" command.
 			if err != nil {
 				return
 			}
-			err = kube.NewCRManager(clientset, clientsetext).Init(ctx)
+
+			err = starboard.NewCRManager(clientset, clientsetext).Init(ctx, options, outWriter)
 			return
 		},
 	}
+	cmd.Flags().BoolVar(&options.DryRun, "dry-run", false, "Only print the object that would be sent, without sending it.")
```

Comment:
If I look at the output of `kubectl create deployment -h` the --dry-run flag accepts 3 values. In our case it's a boolean. Can we make it consistent?

--dry-run='none': Must be "none", "server", or "client". If client strategy, only print the object that would be
sent, without sending it. If server strategy, submit server-side request without persisting the resource.